client/src/auth/index.tsx

```diff
@@ -21,7 +34,11 @@ export class AuthService {
       audience: process.env.REACT_APP_AUTH0_AUDIENCE || '',
     });
     this._isAuthenticated = await this.client.isAuthenticated();
-    this.user = await this.client.getUser();
+
+    if (this._isAuthenticated) {
+      const _user = await this.client.getUser();
+      this.user = _user;
+    }
```

Comment:
Can we avoid the use of `_user`? since we're not using it anything else.

Code example:
```typescript
if (this._isAuthenticated) {
  this.user = await this.client.getUser();
}
```